angular/src/app/discover/discover.component.ts

```diff
@@ -20,19 +22,33 @@ export class DiscoverComponent implements OnInit {
       this.getProjects();
     }
 
-    getProjects() {
-      this.discoverService.getProjects().subscribe(
-          (response: Array<any>) => {
-          this.projects = response;
-          },
-          error => {
-          this.error = true;
-          }
-      );
-    }
+  getProjects() {
+    console.log(this.user_id);
```

Comment:
Please remove all console log statements not used for error logging.

Reply:
Got it, I'll remember that for next time!